includes/modules/shipping/usps.php

```diff
@@ -718,6 +718,11 @@ public function quote($method = '')
                 // international
                 $usps_handling_fee = MODULE_SHIPPING_USPS_HANDLING_INT;
             }
+            
+            // -----
+            // Give an observer the opportunity to modify handling costs.
+            //
+            $this->notify('NOTIFY_SHIPPING_USPS_AFTER_HANDLING', array(), $order, $usps_shipping_weight, $shipping_num_boxes, $usps_handling_fee);
```

Comment:
Please update to use 'short array syntax'.

Reply:
Now you got me confused... 
Isn't long array syntax the norm in ZC? Haven't seen a single short array syntax in core code (with notifiers), and this one is in line with all other notifiers in the file...

Reply:
We're migrating towards short-array syntax.